insertSome_andThenSearch.py

```diff
@@ -11,37 +11,37 @@
     namespace="default_keyspace",
 )
 # print(db)
-
+collection_name = "vector_test2"
 # metric will defined which sort strategy to use. I am a little squishy on what 'dimension' is doing here.
-collection = db.create_collection("vector_test", dimension=5, metric="dot_product")
-# "dimension": 5, // TODO: dimension = size of the array that the vectors will have? I think this is a hint to astra on how to organize things.
-# "metric": "cosine" // Slow but no need to normalize the vectors first
-# "metric": "dot_product" // Quick but have to normalize the vectors first
-# "metric": "euclidean" // Good for Clustering
+collection = db.create_collection(collection_name, dimension=5, metric="dot_product")
+# # "dimension": 5, // TODO: dimension = size of the array that the vectors will have? I think this is a hint to astra on how to organize things.
```

Comment:
NIT: The double `#` can cause surprise when trying to uncomment. Do these need to be double-commented?